lib/mastodon/accounts_cli.rb

```diff
@@ -397,19 +397,21 @@ def refresh(username = nil)
         end
 
         say("Refreshed #{processed} accounts#{dry_run}", :green, true)
-      elsif username.present?
-        username, domain = username.split('@')
-        account = Account.find_remote(username, domain)
-
-        if account.nil?
-          say('No such account', :red)
-          exit(1)
-        end
+      elsif username.length() > 0
```

Comment:
I would use `.size` rather than `.length()` (no real reason except consistency with the remaining of the code base.

Reply:
Fixed in https://github.com/mastodon/mastodon/pull/23304/commits/eaf3e79ddb53aeebc94b7e0f66c9974a69e3fb68